.travis.yml

```diff
@@ -1,38 +1,25 @@
-sudo: false
+os:
+  - linux
+  - windows
+  - osx
 
-env:
-  - CXX=g++-4.8
-addons:
-  apt:
-    sources:
-      - ubuntu-toolchain-r-test
-    packages:
-      - g++-4.8
+sudo: false
 
 language: node_js
 
 node_js:
-  - "0.8"
-  - "0.10"
-  - "0.12"
-  - "1"
-  - "2"
-  - "3"
-  - "4"
-  - "5"
-  - "6"
-  - "7"
+  - "8"
+  - "10"
+  - "node"
```

Comment:
What is this version?

Reply:
We should also test `11` for now I think

Reply:
yes, node means the latest node and that presently is 11.6.0.